packages/ui-components/src/Accordion.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
```

Comment:
This whole file could be simplified into:
```
import SUIAccordion from 'semantic-ui-react/dist/commonjs/modules/Accordion';

export const SUIAccordion;
```

Maybe inside the the `index.ts`, we could export some SUI components as-is? The ones that we want to expose in `ui-components`

Reply:
I should have left a comment about this but the reason I made this and SUICard component like these is to later customize the Header, Description, etc. as well as the props in later PRs as they become necessary, but we're currently just using the defaults since it's not really a priority atm. I'm also hesitant to let the Shared styles file get too long.